# possible grippers: 0) dh3, 1) hand1, 2) hand2
throw_node:
  SIMULATION: 0         # use gazebo as robot
  GRIPPER: 1            # gripper used
  RATE: 1000            # hz of throwing traj
  RATE_CLOSE: 1.9       # hz of closure hand
  HAND_DELAY: 0.5       # 0.2 dh3, 0.6 hand2, 0.4 hand1
  HAND_EFFORT_MAX: 0.15 # 0.15 hand2, 0.8 hand1
  Q0_THROW: [1.89582, 0.36526, -0.0403498, -1.80959, -1.75471, 0.8352, -1.26802]
  QF_THROW: [0.729568, 0.222592, -0.672178, -1.69969, -2.50666, 1.98799, -0.44609]
  P0_EST: [0.446489, -0.196457, 0.289419]           # desired center of estimating point
  P0_PICK: [0.452004, -0.16528, 0.030591] # Per bobina
  P0_THROW: [0.272697, 0.364462, 0.310598]
  PF_THROW: [0.791429, -0.00130922, 0.537626]    # throwing point
  RPY0_PICK: [3.01607, -0.0237897, 0.11158]
  RPY0_THROW: [0.463404, 0.973553, -0.60816]
  RPYF_THROW: [1.04328, 0.58188, -0.484891]
  # RPY0_THROW: [1.92647, -3.10564, -1.94921]
  DPF_THROW: [1.2, -0.7, 0.9]              # throwing velocity
  TF_THROW: 0.9
  TF_BRAKE: 0.3
  TF_EST_BRAKE: 0.5
  TF_EST: 6.0
  PF_THROW_EST: [0.468341, 0.306827, 0.426758] #[0.4, 0.1, 0.4] 
  TF_THROW_EST: 1.0
  DPF_THROW_EST: [0, 1.5, 0.9]
  # q_throw_1: [0.894005, -0.621265, -1.22914, -2.63406, 1.44333, 1.41412, 0.615504]
  ee_pose: [0.405936, 0.07971, 0.344933, -0.36646, 0.290721, 0.533455, 0.704708]
  q_throw_2: [1.75216, -0.183465, -0.828613, -2.245, 2.6374, 1.74342, 0.420288]
  ee_pose: [0.319759, 0.56208, 0.475504, -0.263028, 0.280722, 0.630317, 0.674323]